tests/test_javascript.py

```diff
@@ -239,7 +239,7 @@ def test_duplicate(tmpdir):
         content = fd.read()
         assert (
             content.count(
-                'import CounterMetricType from "@mozilla/glean/webext/private/metrics/counter";'  # noqa
+                'import CounterMetricType from "@mozilla/glean/private/metrics/counter";'  # noqa
```

Comment:
Why is this needed?

Reply:
I changed the way to build Glean.js, so that we don't need to duplicate the whole lib for each platform as before. So all the `private/` modules for all platforms are served from the same entry point (aside from Qt, of course). See https://github.com/mozilla/glean.js/pull/768